.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,20 +46,14 @@ Examples:
   * backwards-compatible bug fix
     * nothing about how to use the code has changed
     * nothing about the outcome of the code has changed (though it likely corrected it)
-  * changes to demo app (_/tests/dummy/_)
 * **MINOR**
   * adding functionality in a backwards-compatible manner
     * nothing about how used to use the code has changed but using it in a new way will do new things
     * nothing about the outcome of the code has changed without having to first use it in a new way
-    * addition of new CSS selectors
-    * addition of new `ember-hook` selectors
 * **MAJOR**
   * incompatible API change
     * using the code how used to will cease working
     * using the code how used to will have a different outcome
-    * any changes to CSS selector names
-    * any removal of CSS selectors
-    * any changes to `ember-hook` selectors
     * possibly changes to test helpers (depends on the changes made)
   * any changes to the **_dependencies_** entry in the _package.json_ file
```

Comment:
Why would changes to `dependencies` be a breaking change? Isn't that kinda the whole point of `dependencies`? That the consumer doen't have to know or care about them? 

True, that's not the case when dealing with `ember-cli`, but this is a straight-up node package, nothing to do with `ember-cli`.

Reply:
I have updated the template to remove the mention of dependencies since this is a node package.